axis.py

```diff
@@ -89,26 +89,6 @@ def __init__(self, _db, collection):
             )
             
         self.token = None
-
-    @classmethod
-    def from_environment(cls, _db):
```

Comment:
I removed every `from_environment` call. You can no longer get "the default API" because there is no longer a "default Collection". This is going to break a lot of scripts.